app/components/dialogs/prioritize-build-modal.js

```diff
@@ -31,6 +31,9 @@ export default Component.extend({
         yield record.increasePriority(this.shouldCancelRunningJobs);
         this.flashes.success('The build was successfully prioritized.');
         this.set('isOpen', false);
+
+        const targetBuild = this.build || this.job.build;
+        targetBuild.set('priority', true);
```

Comment:
That may put the app into inconsistent state. For example, if the request have silently failed to update the priority on backend, we'll have it `false` on backend and `true` on frontend. I'd recommend `targetBuild.reload()` to update the real state from backend.

Reply:
Done.

Reply:
Verified as well, as it is working as expected.

Reply:
Did you forget to push your changes?